### ISSUE TITLE
HF-3c calculations fail because no basis set was specified

### ISSUE THREAD

**Issue by ChayaSt** [MolSSI/QCFractal#501]
**Describe the bug**
HF-3c geometry optimization failed with the following error:
```
ComputeError(error_type=unknown, error_message=geomeTRIC run_json error:
Traceback (most recent call last):
  File "/home/chodera/miniconda/envs/qcfractal/lib/python3.7/site-packages/geometric/run_json.py", line 214, in geometric_run_json
    geometric.optimize.Optimize(coords, M, IC, engine, None, params)
  File "/home/chodera/miniconda/envs/qcfractal/lib/python3.7/site-packages/geometric/optimize.py", line 1331, in Optimize
    return optimizer.optimizeGeometry()
  File "/home/chodera/miniconda/envs/qcfractal/lib/python3.7/site-packages/geometric/optimize.py", line 1293, in optimizeGeometry
    self.calcEnergyForce()
  File "/home/chodera/miniconda/envs/qcfractal/lib/python3.7/site-packages/geometric/optimize.py", line 1002, in calcEnergyForce
    spcalc = self.engine.calc(self.X, self.dirname)
  File "/home/chodera/miniconda/envs/qcfractal/lib/python3.7/site-packages/geometric/engine.py", line 873, in calc
    return self.calc_new(coords, dirname)
  File "/home/chodera/miniconda/envs/qcfractal/lib/python3.7/site-packages/geometric/engine.py", line 865, in calc_new
    raise QCEngineAPIEngineError("QCEngineAPI computation did not execute correctly. Message: " + ret["error"]["error_message"])
geometric.errors.QCEngineAPIEngineError: QCEngineAPI computation did not execute correctly. Message: QCEngine Input Error: Traceback (most recent call last):
  File "/home/chodera/miniconda/envs/qcfractal/lib//python3.7/site-packages/psi4/driver/json_wrapper.py", line 217, in run_json
    json_data = run_json_qcschema(copy.deepcopy(json_data), clean)
  File "/home/chodera/miniconda/envs/qcfractal/lib//python3.7/site-packages/psi4/driver/json_wrapper.py", line 297, in run_json_qcschema
    core.set_global_option("BASIS", json_data["model"]["basis"])
TypeError: set_global_option(): incompatible function arguments. The following argument types are supported:
    1. (arg0: str, arg1: list) -> bool
    2. (arg0: str, arg1: int) -> bool
    3. (arg0: str, arg1: float) -> bool
    4. (arg0: str, arg1: str) -> bool

Invoked with: 'BASIS', None

, extras=None)
```

**Expected behavior**
HF-3c calculations should not require a basis set specification. In this [dataset](https://github.com/openforcefield/qca-dataset-submission/blob/master/2019-11-27-kinase-inhibitor-optimization/02_create_optimization_input.py#L100), no basis was specified. It should be able to run.

**Comment by dgasmith** [comment]
Interesting, thank you for bringing this up. This may require a QCEngine patch to get it working. Will get on it ASAP Monday.

**Comment by dgasmith** [comment]
Fixes in https://github.com/MolSSI/QCEngine/pull/190.

**Comment by j-wags** [comment]
FYI/no response needed: I also saw that error in the logs while I was debugging QCF on the Pacific Research Platform on Friday.

**Comment by dgasmith** [comment]
Closed by MolSSI/QCEngine#190.
